index.html

```diff
@@ -33,6 +33,105 @@ <h1 class="header2">I’m Iptisam
+                <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="Lang">
+                    <li class="HTML"><a href="#">html</a></li>
+                    <li class="css"><a href="#">CSS</a></li>
+                    <li class="Java"><a href="#">Javascript</a></li>
+                </ul>
+                <ul class="box">
+                <li class="button"><a href="#">See Project</a></li>
+                </ul>
+            </div>
+
+            <div class="container2">
+                <div class="rect2"></div>
+                <h3 class="Proj2">Multi-Post Stories</h3>
+                <ul class="list2">
+                    <li class="frame1"><img src="Images/Frame2.png" alt="frame"></li>
+                </ul>
+                <p class="text2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="Lang1">
+                    <li class="HTML1"><a href="#">html</a></li>
+                    <li class="css1"><a href="#">CSS</a></li>
+                    <li class="Java1"><a href="#">Javascript</a></li>
+                </ul>
+                <ul class="box1">
+                <li class="button1"><a href="#">See Project</a></li>
+                </ul>
+            </div>
+
+            <div class="container3">
+                <div class="rect3"></div>
+                <h4 class="Proj3">Tonic</h4>
+                <ul class="list3">
+                    <li class="frame2"><img src="Images/Frame2.png" alt="frame"></li>
+                </ul>
+                <p class="text3">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="Lang2">
+                    <li class="HTML2"><a href="#">html</a></li>
+                    <li class="css2"><a href="#">CSS</a></li>
+                    <li class="Java2"><a href="#">Javascript</a></li>
+                </ul>
+                <ul class="box2">
+                <li class="button2"><a href="#">See Project</a></li>
+                </ul>
+            </div>
+
+            <div class="container4">
+                <div class="rect4"></div>
+                <h5 class="Proj4">Multi-Post Stories</h5>
+                <ul class="list4">
+                    <li class="frame3"><img src="Images/Frame2.png" alt="frame"></li>
+                </ul>
+                <p class="text4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="Lang3">
+                    <li class="HTML3"><a href="#">html</a></li>
+                    <li class="css3"><a href="#">CSS</a></li>
+                    <li class="Java3"><a href="#">Javascript</a></li>
+                </ul>
+                <ul class="box3">
+                <li class="button3"><a href="#">See Project</a></li>
+                </ul>
+            </div>
+            <div class="container5">
```

Comment:
- [ ] Kindly separate your about section to your projects section by ensuring the container tag fo the about section is `section` tag.

`hint`
```suggestion
            <section class="container5">
```